### PR TITLE
update photo data in "Capture.cs" and updated codes 

### PR DESCRIPTION
**Pull request by Sara-webaplication**
* Startup.cs: updated codes
* Capture.cs: updated codes base william codes
* AdminController.cs: updated codes
* maple-leaf-888807_640.jpg:
* photo_points.Tests.csproj:
* AdminReviewServices.cs: updated codes
* IAdminReviewServices.cs:updated codes
* FakeAdminReviewRepository.cs: updated photo data and codes

### CODE REVIEW

**Comment by Sara-webaplication** [comment]
I updated codes base Eric and William's codes that changed and also I added code to convert photo to byte.

File: photo_points/Repositories/FakeAdminReviewRepository.cs [hunk 1]
```diff
@@ -56,8 +69,9 @@ public Capture GetCapture(long id)
 
         public IEnumerable<Capture> GetAllUnapproved()
         {
-            return captures.Where(a => a.Approved == true);
-            //Needs Willie's updated Capture model with approved property
+           // return captures.Where(a => a.approve == true);
+         return captures.Where(a => a.approval == Capture.ApprovalType.Approve);
```
Comment (Bansenauer-Cascadia) [inline photo_points/Repositories/FakeAdminReviewRepository.cs:73]: This seems to be returning all of the Approved photos rather than the "Unapproved"

**Review by Bansenauer-Cascadia — COMMENTED** [review]
Good progress Sara - your getting there. See my note in the code.

**Review by weberwilliamb — CHANGES_REQUESTED** [review]
Looks like you need to pull down the develop branch into your code.

File: photo_points/Repositories/FakeAdminReviewRepository.cs (hunk 1 above)
Comment (Sara-webaplication) [reply photo_points/Repositories/FakeAdminReviewRepository.cs:73]: that is Eric codes i just  updated Name base changes

**Review by sruble — APPROVED** [review]
thanks Sara, it looks good.

**Review by weberwilliamb — APPROVED** [review]
Sweet! Looks great!

**Review by MaxIgnatyev — APPROVED** [review]

File: photo_points/Repositories/FakeAdminReviewRepository.cs (hunk 1 above)
Comment (Sara-webaplication) [reply photo_points/Repositories/FakeAdminReviewRepository.cs:73]: i will change it to pending

**Pull request merged**